### PR TITLE
Moe/translations

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: app/components/settings/backupSeed/BackupSeedIntro.tsx [hunk 1]
```diff
@@ -8,36 +8,48 @@ import * as noComputer from "../../../assets/img/seed_backup/no_computer.svg";
...
-      </Col>
-    </Row>
-  </>
+export const BackupSeedIntro = injectIntlHelpers<IBackupSeedIntroProps>(
+  ({ intl: { formatIntlMessage }, onBack, onNext }) => (
+    <>
+      <Row className="mb-4 text-center">
+        <StepCard
+          img={write}
+          text={formatIntlMessage("settings.backup-seed-intro.write-all-words")}
```
Comment (krzkaczor) [inline app/components/settings/backupSeed/BackupSeedIntro.tsx]: couldnt you use formatted message as well here after changing typings?

File: app/components/settings/backupSeed/BackupSeedVerify.tsx [hunk 2]
```diff
@@ -44,20 +46,21 @@ const getRandomNumbers = (numbers: number, range: number): number[] => {
...
+  constructor(props: IBackupSeedVerifyProps & IIntlProps) {
+    super(props);
     const wordsToCheck = getRandomNumbers(WORDS_TO_VERIFY, props.words.length);
 
     this.state = {
       verificationWords: wordsToCheck.map(number => ({ number: number, word: "" })),
     };
   }
-
+  private formatIntlMessage = this.props.intl.formatIntlMessage;
```
Comment (krzkaczor) [inline app/components/settings/backupSeed/BackupSeedVerify.tsx]: introduce some new lines here and keep field assignments together (there is already displayName)

File: app/components/settings/backupSeed/BackupSeedIntro.tsx (hunk 1 above)
Comment (Moejoe90) [reply app/components/settings/backupSeed/BackupSeedIntro.tsx]: :+1:

File: app/components/settings/backupSeed/BackupSeedVerify.spec.tsx [hunk 3]
```diff
@@ -66,15 +75,17 @@ describe("<BackupSeedVerify />", () => {
         { number: 3, word: words[3] },
       ],
     };
-
     component.setState(state);
 
     expect(component.find(tid("seed-verify-invalid-msg"))).to.have.lengthOf(0);
+    component.update();
```
Comment (krzkaczor) [inline app/components/settings/backupSeed/BackupSeedVerify.spec.tsx]: is it really needed here?

Comment (Moejoe90) [reply app/components/settings/backupSeed/BackupSeedVerify.spec.tsx]: Good catch, this is not needed at all at the moment

**Pull request merged**